### PR TITLE
added documentation of features in Cake for Rider version 1.0.0

### PR DESCRIPTION
**Pull request by nils-a**

### CODE REVIEW

File: input/docs/integrations/editors/rider/templates.md
Comment (ecampidoglio) [inline input/docs/integrations/editors/rider/templates.md:65]: **nitpick**: shouldn't it be "including _a_ `.Does` body"?

File: input/docs/integrations/editors/rider/templates.md
Comment (ecampidoglio) [inline input/docs/integrations/editors/rider/templates.md:31]: **suggestion**: I would format the word "_addin_" as `#addin`.

File: input/docs/integrations/editors/rider/templates.md
Comment (ecampidoglio) [inline input/docs/integrations/editors/rider/templates.md:40]: **suggestion**: I would format the word "_load_" as `#load`.

File: input/docs/integrations/editors/rider/templates.md
Comment (ecampidoglio) [inline input/docs/integrations/editors/rider/templates.md:49]: **suggestion**: I would format the word "_reference_" as `#reference`.

File: input/docs/integrations/editors/rider/templates.md
Comment (ecampidoglio) [inline input/docs/integrations/editors/rider/templates.md:54]: **suggestion**: I would format the word "_tool_" as `#tool`.

**Review by ecampidoglio — APPROVED** [review]
LGTM with a few suggestions.

**Comment by nils-a** [comment]
Cake for Rider 1.0.0 has been released 🎉 
I will merge this PR, once it has been approved by JetBrains

**Review by gep13 — APPROVED** [review]
LGTM!

**Review by augustoproiete — APPROVED** [review]
LGTM!

image

**Pull request merged**
